patches/node/fix_make_sharedarraybuffer_optional_in_freeze_intrinsics_js.patch

```diff
@@ -0,0 +1,39 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Milan Burda <milan.burda@gmail.com>
+Date: Sat, 3 Jun 2023 23:18:01 +0200
```

Comment:
Have you tried upstreaming this? I'd want to know whether Node.js would take it upstream before floating this patch.

Reply:
https://github.com/nodejs/node/pull/49482